packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -203,6 +203,7 @@ def reduce_logging():
     logging.getLogger('test_util.marathon').setLevel(start_log_level)
 
 
+@pytest.mark.xfail(reason="DCOS-19366 - xfail test_vip on DCOS 1.9 due to high flakiness")
 @pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
 def test_vip(dcos_api_session, reduce_logging):
```

Comment:
Instead of disabling test_vip, can we just copy over the `test_vip` changes from 1.10/1.11 into 1.9? This is pretty critical to verify a lot of the networking pieces so would rather not have it disabled.

Reply:
@urbanserj ?? ^^

Reply:
@asridharan - I and @urbanserj  had a discussion before we embarked on this. It will be good on the same page.

For the context, even with 1.9 (test_vip flake being mute) our CI / Test infrastructure didn't succeed in single go.

Reply:
@asridharan test_vip was rewritten almost from scratch in 1.10 several times. it’s not possible to backport only my changes now :(

Reply:
can't we just copy the `test_vip` functions from 1.10 into this branch, along with the helper functions? Or is it that the dependencies have changed dramatically in 1.10?

Reply:
I am just worried that once `test_vip` is disabled we won't have a single networking test of some real importance in 1.9. I understand that these tests in their current form are pretty much useless (since they hardly pass), but trying to take this opportunity to see we can have the effort we put in 1.10 and 1.11 pay some dividends here.

Reply:
It's not possible just copy the code of `test_vip` from 1.10 into this branch. Our integration test infrastructure was changed dramatically. if I understand this patch correctly the test will be run any way, but failures will not affect status of the build. You can check `test_vip` status by hands.

Reply:
I see. Unfortunate. I agree in its current form these tests don't add much value and setting it to `xfail` makes sense. @orsenthil do we not have the same harnesses that we had in 1.10 for 1.9? If not we need to port them 1.9 at the earliest so that we can copy over the 1.10 tests to 1.9.

Reply:
@asridharan - by test harness, it was the software test harness. The code got refactored into external libraries and things are not the same.
The CI infrastructure for both 1.9 and 1.10 and `master` are same. 

With `xfail` we are still good that these are exercised, but wont block anyone because of inherent high flakiness.